test/InputSpec.js

```diff
@@ -22,6 +22,14 @@ describe('<Input />', () => {
     });
   });
 
+  context('#text with value', () => {
+    it('renders an input with value', () => {
```

Comment:
thanks for this, could we add another assertion for the `array` type?

Reply:
like what we expect to appear when you add 2 values in the `value` prop

Reply:
I saw that there are another tests that check 'select' element that already validate it so I didn't add test for this case.

Reply:
I think we should test that at least it doesn't break

Reply:
I've added test case